Dockerfile

```diff
@@ -8,25 +8,36 @@ RUN --mount=source=.,target=/mnt,type=bind \
                                                vim python3-all pciutils binutils tcpdump \
                                                rsyslog iptables ssl-cert openssl ca-certificates \
                                                curl nmap hydra hping3 iperf3 d-itg socat \
- && apt install -y /mnt/misc/shellinabox_2.21.1_amd64.deb \
- && echo "root:hackinsdn" | chpasswd \
+ && apt install -y /mnt/misc/shellinabox_2.21.1_amd64.deb
+
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN --mount=source=.,target=/mnt,type=bind \
+    cp -vr /mnt/conf/* / \
+ && curl -L --insecure https://trex-tgn.cisco.com/trex/release/v3.05.tar.gz | tar -xz -C /opt/trex
+
+RUN echo "root:hackinsdn" | chpasswd \
  && sed -i 's/#ListenAddress 0.0.0.0/ListenAddress 127.0.0.1/g' /etc/ssh/sshd_config \
- && sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/g' /etc/ssh/sshd_config \
- && cp -r /mnt/src /var/www/html/secflood \
- && cp -vr /mnt/conf/* / \
- && curl -L --insecure https://trex-tgn.cisco.com/trex/release/v3.05.tar.gz | tar -xz -C /opt/trex \
- && install --mode 0755 --owner root /mnt/scripts/trex /usr/local/bin/ \
- && mkdir -p /var/lib/sqlite \
+ && sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/g' /etc/ssh/sshd_config
+
+RUN --mount=source=.,target=/mnt,type=bind \
+    cp -r /mnt/src /var/www/html/secflood \
+ && install --mode 0755 --owner root /mnt/scripts/trex /usr/local/bin/
+
+RUN mkdir -p /var/lib/sqlite \
  && sqlite3 /var/lib/sqlite/secflood.db < /var/www/html/secflood/assets/database.sql \
- && chown www-data /var/lib/sqlite/secflood.db \
- && a2dissite 000-default \
+ && chown www-data /var/lib/sqlite/secflood.db
+
+RUN a2dissite 000-default \
  && a2enmod proxy_http \
  && a2enmod rewrite \
  && a2enmod ssl \
- && a2ensite secflood \
- && mv /etc/shellinabox/options-enabled/00+Black\ on\ White.css /etc/shellinabox/options-enabled/00_Black\ on\ White.css \
- && mv /etc/shellinabox/options-enabled/00_White\ On\ Black.css /etc/shellinabox/options-enabled/00+White\ On\ Black.css \
- && install --mode 0755 --owner root /mnt/docker-entrypoint.sh /docker-entrypoint.sh \
- && rm -rf /var/lib/apt/lists/*
+ && a2ensite secflood
+
+RUN mv /etc/shellinabox/options-enabled/00+Black\ on\ White.css /etc/shellinabox/options-enabled/00_Black\ on\ White.css \
+ && mv /etc/shellinabox/options-enabled/00_White\ On\ Black.css /etc/shellinabox/options-enabled/00+White\ On\ Black.css
```

Comment:
Here another suggestion: I believe we should have one separated step for loading the sqlite database, which is more likely to be throw errors; for the remain steps I dont see advantages on separating them. I suggest grouping them all together.

Reply:
I suggest keeping it as it is because `mkdir` and `chown` commands are `sqlite` related and unlikely to throw errors. I also separated the `apache` and `shellinabox` configuration commands in their 'own' groups.